libs/blocks/ost/ost.js

```diff
@@ -2,23 +2,22 @@ import { loadScript, loadStyle } from '../../utils/utils.js';
 
 const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
 const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
-const OST_SCRIPT_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/index.js';
-const OST_STYLE_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/index.css';
+const OST_SCRIPT_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.js';
+const OST_STYLE_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.css';
+
+const ENVIRONMENT = 'PROD';
+const WCS_API_KEY = 'wcms-commerce-ims-ro-user-cc';
+const AOS_API_KEY = 'wcms-commerce-ims-user-prod';
+const CHECKOUT_CLIENT_ID = 'creative';
+
+const searchParameters = new URLSearchParams(window.location.search);
+const token = searchParameters.get('token');
```

Comment:
This is more of a workaround for testing purposes and should be documented here at the very least with a note imo.

Reply:
@honstar commented the line.